scripts/python/phdf_diff.py

```diff
@@ -141,7 +118,7 @@ def addPath():
     if not brief and not quiet:
         print('____Comparing on a per variable basis with tolerance %.16g'%tol)
     breakOut = False
-    oneTenth = f0.TotalCells/10
+    oneTenth = f0.TotalCells//10
```

Comment:
Was this intentional?

Reply:
Yes, this makes sure the result is an integer (python 3).